hw1_iphone/styles/styles .css

```diff
@@ -0,0 +1,335 @@
+  left: 68px;
+}
+
+.front-camera_top {
+  position: absolute;
+  border-radius: 50%;
+  border: 0.2rem solid #1c1e14;
+  background: #0f0f0e;
+  top: 3px;
+  left: 110px;
+}
+
+.microphone {
+  position: absolute;
+  top: 20px;
+  right: 87px;
+  width: 42px;
+  height: 3px;
+  border-radius: 0.2rem;
+  background: #1d1312;
+  border: 1px solid #9f9796;
+}
+
+.bottom-btn {
+  position: absolute;
+  border-radius: 50%;
+  border: 3px solid transparent;
+  background: linear-gradient(white, white) padding-box,
+    linear-gradient(to top left, #3c3628, var(--white)) border-box;
+  top: 533px;
+  left: 124px;
+  width: 45px;
+  height: 43px;
+  cursor: pointer;
+  opacity: var(--opacity-default: );
+}
+
+.top-bar {
+  position: absolute;
+  width: 251px;
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row wrap;
+  align-items: center;
+  margin: 2px 5px;
+  opacity: var(--opacity-default: );
+}
+
+.signal-icons {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-flow: row wrap;
+}
+
+.screen:hover .signal-icons {
+  visibility: visible;
+}
+
+.provider {
+  font-size: 9px;
+  line-height: 0px;
+  padding: 0 2px;
+  font-family: SF-Pro-Medium;
+}
+
+.top-btn {
+  position: absolute;
+  top: 5px;
+  right: 116px;
+  width: 24px;
+  height: 5px;
+  border-radius: 0.2rem;
+  background: var(--bg-color-default: );
+  border: 1px solid var(--text-color-default);
+  opacity: 0.3;
+  cursor: pointer;
+}
+
+.top-btn:hover {
+  top: 7px;
+  transition: var(--transition-duration);
+}
+
+.date {
+  text-align: center;
+  font-size: 13px;
+  font-family: SF-Pro-Bold;
+  margin-top: 0;
+  opacity: 0.7;
+}
+
+.screen:hover .date {
+  visibility: visible;
+}
+
+.screen:hover .battery {
+  visibility: visible;
+}
+
+.time {
+  font-size: 56px;
+  font-weight: initial;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 4px;
+  opacity: var(--opacity-default: );
+  font-family: SF-Pro-Thin;
+}
+
+.screen:hover .time {
+  visibility: visible;
+}
+
+.slider {
+  display: flex;
+  margin-top: 250px;
+  justify-content: center;
+  align-items: center;
+  opacity: 0.7;
+  cursor: pointer;
+}
+
+.screen:hover .slider {
+  visibility: visible;
+}
+
+.slider:hover {
+  transition: 0.2s;
+  margin-left: 7px;
+}
+
+.slider-text {
+  font-size: 17px;
+  font-weight: 300;
+  margin-left: 7px;
+  font-family: SF-Pro-Medium;
+  color: transparent;
+  -webkit-background-clip: text;
+  background-clip: text;
+  background-image: linear-gradient(
+    90deg,
+    rgb(43, 43, 43) 27%,
+    rgb(20, 20, 20) 51%,
+    rgb(198, 202, 203) 70%
+  );
+}
+
+.screen:hover .slider-text {
+  visibility: visible;
+}
+
+.bottom-bar {
+  display: flex;
+  margin-left: 120px;
+  flex-direction: row;
+  justify-content: space-between;
+  align-content: space-between;
+  align-items: center;
+}
+
+.camera-btn {
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  margin-bottom: -48px;
```

Comment:
I'm pretty sure that you can do it without negative margins.
Typically negative margin means "it is a hack"